deploy/docker-compose.yaml

```diff
@@ -31,54 +59,82 @@ services:
     depends_on:
       - nuoadmin1
       - sm
-    command: [ "nuodocker", "start", "te", "--db-name", "hockey", "--server-id", "nuoadmin1" ]
+    command:
+      [
+        "nuodocker",
+        "start",
+        "te",
+        "--db-name",
+        "hockey",
+        "--server-id",
+        "nuoadmin1",
+      ]
   influxdb:
-    image: influxdb:1.8
+    image: influxdb:2.7
     ports:
       # The API for InfluxDB is served on port 8086
       - "8086:8086"
       - "8082:8082"
     volumes:
       - ../deploy/initdb.sh:/docker-entrypoint-initdb.d/initdb.sh
+    environment:
+      DOCKER_INFLUXDB_INIT_MODE: $INFLUXDB_MODE
+      DOCKER_INFLUXDB_INIT_USERNAME: $INFLUXDB_USERNAME
+      DOCKER_INFLUXDB_INIT_PASSWORD: $INFLUXDB_PASSWORD
+      DOCKER_INFLUXDB_INIT_ORG: $INFLUXDB_ORG
+      DOCKER_INFLUXDB_INIT_RETENTION: $INFLUXDB_RETENTION
+      DOCKER_INFLUXDB_INIT_BUCKET: $INFLUXDB_BUCKET
+      DOCKER_INFLUXDB_INIT_ADMIN_TOKEN: $INFLUXDB_ADMIN_TOKEN
+
   nuocd-sm:
     image: nuodb/nuodb-collector:latest
```

Comment:
I thought that you had some changes in the `nuocollector` Docker image that supports InfluxDB2.
How come the automated test pass before the new image is released?

Reply:
The only change in Docker image of nuocollector is update of telegraf version. Since the output plugin inside dynamic folder is updated by insights, the test are passing.